scalopus_tracing/include/scalopus_tracing/trace_macro.h

```diff
@@ -66,4 +66,9 @@
 #define TRACE_MARK_EVENT_PROCESS(name) TRACE_MARK_EVENT_NAMED_ID(PROCESS, name, SCALOPUS_TRACKED_TRACE_ID_STRING(name))
 #define TRACE_MARK_EVENT_THREAD(name) TRACE_MARK_EVENT_NAMED_ID(THREAD, name, SCALOPUS_TRACKED_TRACE_ID_STRING(name))
 
+// Macro to set a counter series value.
+#define TRACE_COUNT_SERIES(name, series_name, value) TRACE_COUNT_SERIES_EVENT_NAMED(name "/" series_name, value)
+
+// Macro to set a counter value, just one series called 'count'.
+#define TRACE_COUNT(name, value) TRACE_COUNT_SERIES(name, "", value)
```

Comment:
If I understand correctly, you'll get `name/` in this case. Is that what we'll see in the .json file and the viewer?

Reply:
Your two last commits went in before I sent my review. nvm